src/pages/search.jsx

```diff
@@ -0,0 +1,58 @@
+import React, {useState} from 'react';
+import axios from "axios";
+import spinningWorld from "../assets/spinning_globe.png"
+import CountryInformationComponent from "../assets/components/country_information_component.jsx";
+
+
+    const Search = () => {
+
+        const [searchResult, setSearchResult] = useState([])
```

Comment:
`searchResult` zal uiteindelijk een object worden en nooit een array. Als je een default waarde van `undefined` gebruikt dan heb je een duidelijke discriminator, om te bepalen of het component wel/niet getoond moet worden
```suggestion
        const [searchResult, setSearchResult] = useState();
```